pkgs/applications/graphics/mcomix3/default.nix

```
@@ -55,20 +57,22 @@ python3.pkgs.buildPythonApplication rec {
 
   postInstall = ''
     rmdir $libdir/mcomix/mcomix
-    cp man/* $out/share/man/man1/
+    installManPage man/*
+    mv $out/share/man/man1/mcomix.1 $out/share/man/man1/${pname}.1
```

```suggestion
    mv man/mcomix.1 man/${pname}.1
    installManPage man/*
```

I am not sure if this messes with compression or anything else.